screens/join.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../css/style.css" />
+    <link rel="stylesheet" href="../css/pages/login/login.css" />
+    <title>판다마켓 - join</title>
+  </head>
+  <body>
+    <div class="total-container">
+      <main class="loginContainer">
+        <a href="../index.html" class="login-logo">
+          <img src="../img/logo.png" / >
+        </a>
+        <div class="login-form-container">
+          <form class="login-form">
+            <label for="login-email"> 이메일 </label>
```

Comment:
❗️ 수정요청
공백이 들어가 있으면 정확하게 디자인을 구현하기 어렵습니다~

```suggestion
            <label for="login-email">이메일</label>
```